src/dmd/dsymbolsem.d

```diff
@@ -1145,13 +1145,20 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             sc.protection = imp.protection;
             for (size_t i = 0; i < imp.aliasdecls.dim; i++)
             {
-                bool flag;
                 AliasDeclaration ad = imp.aliasdecls[i];
                 //printf("\tImport %s alias %s = %s, scope = %p\n", toPrettyChars(), aliases[i].toChars(), names[i].toChars(), ad._scope);
                 Dsymbol sym = imp.mod.search(imp.loc, imp.names[i] /*, IgnorePrivateImports */);
                 if (sym)
                 {
-                    flag = true;
+                    // Deprecated in 2018-01.
+                    // Change to error in 2019-01 by deleteting the following 3 lines and uncommenting
```

Comment:
This comment needs to be updated to "following 5 lines", or wordsmithed in a way to where the number of lines doesn't matter.

Reply:
Oh, missed that, sorry. Thanks for the awesome attention to details. I addressed it now.